universal-login-web3/src/services/UIController.ts

```diff
@@ -65,6 +65,9 @@ export class UIController {
   }
 
   showError(errorMessage?: string) {
```

Comment:
Should we ignore it? Maybe we should resend requsest in this case or at least console this error?